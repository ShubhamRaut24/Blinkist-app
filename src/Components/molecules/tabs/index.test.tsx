import React from 'react'
import Tabs from '.';
import renderer from 'react-test-renderer';



const tabData = [
    { 
      'value': 'category',
      'label': 'Explore by category',
    },
    { 
      'value': 'recent',
      'label': 'recently added'
    },
    { 
        'value': 'popular',
        'label': 'See popular titles'
      }
]

  const handleChange = (event: React.SyntheticEvent, newValue: string) => {}
   

test('renders Tab', () => {
    const component = renderer.create(
    <Tabs textColor= 'primary' indicatorColor= 'success' tabData= {tabData} value="recent"  onChange={handleChange}/>
    );
  
  expect(component).toMatchSnapshot();
});
