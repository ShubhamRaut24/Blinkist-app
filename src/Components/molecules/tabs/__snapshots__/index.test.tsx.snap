// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Tab 1`] = `
<div
  className="MuiBox-root css-1g37473"
>
  <div
    autoCapitalize="none"
    className="MuiTabs-root css-1ujnqem-MuiTabs-root"
  >
    <div
      className="MuiTabs-scroller MuiTabs-fixed css-jpln7h-MuiTabs-scroller"
      onScroll={[Function]}
      style={
        Object {
          "marginBottom": -0,
          "overflow": "hidden",
        }
      }
    >
      <div
        aria-label="wrapped label tabs example"
        aria-orientation={null}
        className="MuiTabs-flexContainer css-heg063-MuiTabs-flexContainer"
        onKeyDown={[Function]}
        role="tablist"
      >
        <button
          aria-selected={true}
          className="MuiButtonBase-root MuiTab-root MuiTab-textColorPrimary Mui-selected css-1fjr9ps-MuiButtonBase-root-MuiTab-root"
          disabled={false}
          onBlur={[Function]}
          onClick={[Function]}
          onContextMenu={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          role="tab"
          tabIndex={0}
          type="button"
        >
          Explore by category
          <span
            className="MuiTabs-indicator css-lafep3-MuiTabs-indicator"
            style={Object {}}
          />
        </button>
        <button
          aria-selected={false}
          className="MuiButtonBase-root MuiTab-root MuiTab-textColorPrimary css-1fjr9ps-MuiButtonBase-root-MuiTab-root"
          disabled={false}
          onBlur={[Function]}
          onClick={[Function]}
          onContextMenu={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          role="tab"
          tabIndex={-1}
          type="button"
        >
          recently added
        </button>
        <button
          aria-selected={false}
          className="MuiButtonBase-root MuiTab-root MuiTab-textColorPrimary css-1fjr9ps-MuiButtonBase-root-MuiTab-root"
          disabled={false}
          onBlur={[Function]}
          onClick={[Function]}
          onContextMenu={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          role="tab"
          tabIndex={-1}
          type="button"
        >
          See popular titles
        </button>
      </div>
    </div>
  </div>
</div>
`;
